lib/Meta/Metas.php

```diff
@@ -125,4 +144,72 @@ private function findByTitle(string $text): ?MetaEntry
 
         return null;
     }
+
+    /** @return array<string, LinkTarget> */
+    public function getLinkTargets(): array
+    {
+        return $this->linkTargets;
+    }
+
+    public function setLinkTarget(string $name, LinkTarget $linkTarget): void
+    {
+        if (array_key_exists($name, $this->linkTargets)) {
+            $this->errorManager->warning('Duplicate anchor ".. _' . $linkTarget->getName() . '" found.');
+            $i = 2;
+            while (array_key_exists($name . '-' . $i, $this->linkTargets)) {
+                $i++;
+            }
+
+            $name .= '-' . $i;
+            $linkTarget->setName($name);
+            $linkTarget->setDuplicate(true);
+        }
+
```

Comment:
This part of the code is reached during the failing test, hope this helps.

Reply:
ty